src/dotnet-interactive-vscode/src/interactiveNotebook.ts

```diff
@@ -5,6 +5,8 @@ import { RawNotebookCell } from "./interfaces";
 import { trimTrailingCarriageReturn } from './utilities';
 
 let languageAliases = new Map<string, string>();
+// polyglot language cell type
+languageAliases.set('dotnet-interactive', 'dotnet-interactive');
```

Comment:
a polyglot cell targets the .NET kernel itself (composite), right?

Reply:
Currently backed this part out, so the only thing this PR allows you to do is set the language mode on a text file to `.NET Interactive Script` and it'll get the polyglot coloring.